### PR TITLE
Pull out helper function to ensure valid prompt

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: tests/installation/isosize.pm [hunk 1]
```diff
@@ -11,7 +11,12 @@ sub run {
     if (!$size || !$max || $size > $max) {
         $result = 'fail';
     }
-    bmwqemu::diag("check if actual iso size $size fits $max: $result");
+    if (!defined $size) {
```
Comment (nilxam) [inline tests/installation/isosize.pm]: $size actually is defined earlier above anyway, use if (!$size) or unless here

**Comment by nilxam** [comment]
famous barewards issue :)

File: tests/installation/isosize.pm (hunk 1 above)
Comment (okurz) [inline tests/installation/isosize.pm]: The thing is: I want to differentiate between size being zero or not defined at all. That is why I use the explicit "if !defined"

**Comment by okurz** [comment]
kinda fixed, don't really understand the difference on sub definition, yet. good enough for now :-)

**Comment by coolo** [comment]
become_root should just do it IMO. just as activate 'user-console'

**Comment by okurz** [comment]
Yes, what I did so far is a simple refactoring, i.e. not changing any behaviour, just giving an explicit name for what was previously not obvious paired with a comment with typo. You could grab my change and move the "ensure…prompt" into another function if you are sure where it should be, e.g. "become_root". I don't know why it has been used at some places but not at others or why it should be necessary at all.

**Comment by coolo** [comment]
I don't know either - my assumption is: just not needle required for the prompt :)

**Comment by coolo** [comment]
I'm fine with taking your extract function step. But I don't really agree with the name. The function does not ensure anything, it only types something. How about s,ensure,set, ?

**Comment by okurz** [comment]
I named it "ensure" because this is the intended effect: The prompt is ensured to be "proper". Calling it "set" would be ok but then a comment would be necessary why it is necessary to set it. Of course, we could call it "set_needleable_prompt" which makes it also more obvious :-)

**Comment by coolo** [comment]
set_standard_prompt and set_root_prompt ?

**Comment by coolo** [comment]
took the alternative

**Pull request closed without merging**